tests/test_accessibility.py

```diff
@@ -0,0 +1,66 @@
+"""
+Tests if selected pages of the website are accessible when rendered,
+according to WCAG standards
+"""
+import multiprocessing
+from urllib.request import urlopen
+
+import pytest
+from axe_selenium_python import Axe
+from flask import url_for
+
+multiprocessing.set_start_method("fork")  # Required on macOSX
+
+
+@pytest.mark.app(debug=False)
+def test_app(app):
+    assert not app.debug, "Ensure the app not in debug mode"
+
```

Comment:
make spacing between methods common and classes common

Reply:
Spacing made consistent